### ISSUE TITLE
name

### ISSUE THREAD

**Issue by RaliVault** [M3351AN/TokiKiriMenu#1]
this repo's name is 時霧.
this is kanji right?
are you jp?

**Comment by M3351AN** [comment]
No, I'm Mongolian, but I know a little bit of Chinese and Japanese because I studied in China
